src/platforms/android/common/proguard.mdx

```diff
@@ -51,6 +51,13 @@ defaults.project=your-project
 defaults.org=your-org
 auth.token=YOUR_AUTH_TOKEN
 ```
+
+<Note>
+
+Values inside `sentry.properties` file should not be wrapped around quotes, you should only replace the placeholders. Otherwise, it may lead to authentication or resource errors.
```

Comment:
```suggestion
Values inside `sentry.properties` file should not be wrapped around quotes; you should only replace the placeholders. Otherwise, it may lead to authentication or resource errors.
```